src/Compilers/VisualBasic/Portable/Binding/Binder_Latebound.vb

```diff
@@ -70,7 +70,9 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Dim result = New BoundLateMemberAccess(node, name, containerType, receiver, boundTypeArguments, LateBoundAccessKind.Unknown, objType)
 
-            If receiver IsNot Nothing AndAlso Not receiver.WasCompilerGenerated AndAlso result.Syntax Is receiver.Syntax Then
+            If receiver IsNot Nothing AndAlso
+               ((Not receiver.WasCompilerGenerated AndAlso result.Syntax Is receiver.Syntax) OrElse
+                (receiver.WasCompilerGenerated AndAlso receiver.Syntax.Kind = SyntaxKind.ObjectCreationExpression)) Then
```

Comment:
>eceiver.Syntax.Kind = SyntaxKind.ObjectCreationExpression [](start = 56, length = 57)

Can we use something other than a syntax check? Are we using special bound nodes as receivers in this case?

Reply:
Worked with @AlekseyTs offline to find a better approach.

---
In reply to: [190969490](https://github.com/dotnet/roslyn/pull/27126#discussion_r190969490) [](ancestors = 190969490)